src/components/_tests/ContextBar.test.js

```diff
@@ -0,0 +1,152 @@
+  cloudContext: {
+    cloudProvider: 'GCP',
+    cloudResource: 'terra-dev-cf677740'
+  },
+  auditInfo: {
+    creator: 'testuser123@broad.com',
+    createdDate: '2022-07-18T18:35:32.012698Z',
+    destroyedDate: null,
+    dateAccessed: '2022-07-18T21:44:17.565Z'
+  },
+  runtimeConfig: {
+    machineType: 'n1-standard-1',
+    persistentDiskId: 14692,
+    cloudService: 'GCE',
+    bootDiskSize: 120,
+    zone: 'us-central1-a',
+    gpuConfig: null
+  },
+  proxyUrl: 'https://leonardo.dsde-dev.broadinstitute.org/proxy/terra-dev-cf677740/saturn-eae9168f-9b99-4910-945e-dbab66e04d91/jupyter',
+  status: 'Running',
+  labels: {
+    saturnWorkspaceNamespace: 'general-dev-billing-account',
+    'saturn-iframe-extension': 'https://bvdp-saturn-dev.appspot.com/jupyter-iframe-extension.js',
+    creator: 'testuser123@broad.com',
+    clusterServiceAccount: 'pet-26534176105071279add1@terra-dev-cf677740.iam.gserviceaccount.com',
+    saturnAutoCreated: 'true',
+    clusterName: 'saturn-eae9168f-9b99-4910-945e-dbab66e04d91',
+    saturnWorkspaceName: 'Broad Test Workspace',
+    saturnVersion: '6',
+    tool: 'Jupyter',
+    runtimeName: 'saturn-eae9168f-9b99-4910-945e-dbab66e04d91',
+    cloudContext: 'Gcp/terra-dev-cf677740',
+    googleProject: 'terra-dev-cf677740'
+  },
+  patchInProgress: false
+}
+
+const jupyter1Disk = {
+  id: 14692,
+  googleProject: 'terra-dev-cf677740',
+  cloudContext: {
+    cloudProvider: 'GCP',
+    cloudResource: 'terra-dev-cf677740'
+  },
+  zone: 'us-central1-a',
+  name: 'saturn-pd-c4aea6ef-5618-47d3-b674-5d456c9dcf4f',
+  status: 'Ready',
+  auditInfo: {
+    creator: 'testuser123@broad.com',
+    createdDate: '2022-07-18T18:35:32.012698Z',
+    destroyedDate: null,
+    dateAccessed: '2022-07-18T20:34:56.092Z'
+  },
+  size: 50,
+  diskType: {
+    label: 'pd-standard',
+    displayName: 'Standard',
+    regionToPricesName: 'monthlyStandardDiskPrice'
+  },
+  blockSize: 4096,
+  labels: {
+    saturnWorkspaceNamespace: 'general-dev-billing-account',
+    saturnWorkspaceName: 'Broad Test Workspace'
+  }
+}
+
+describe('getTotalToolAndDiskCostDisplay', () => {
+  it('will get the total cost of all tools in the workspace.', () => {
+    // ARRANGE
+    const runtimes = [jupyter1]
+    const apps = [galaxyRunning]
+    const appDataDisks = [galaxyDisk]
+    const refreshRuntimes = () => ''
+    const location = 'US-CENTRAL1'
+    const locationType = ''
+    const refreshApps = () => ''
+    const workspace = {
+      workspace: {
+        namespace: 'namespace'
+      },
+      namespace: 'Broad Test Workspace'
+
+    }
+    const persistentDisks = [jupyter1Disk, galaxyDisk]
+    render(<ContextBar
+      runtimes={runtimes}
+      apps={apps}
+      appDataDisks={appDataDisks}
+      refreshRuntimes={refreshRuntimes}
+      location={location}
+      locationType={locationType}
+      refreshApps={refreshApps}
+      workspace={workspace}
+      persistentDisks={persistentDisks}
+    />)
```

Comment:
Could this be

```javascript
    render(h(ContextBar, {
      runtimes,
      apps,
      appDataDisks,
      refreshRuntimes,
      location,
      locationType,
      refreshApps,
      workspace,
      persistentDisks
    }))
```
?

Reply:
Taking a different approach - this is no longer relevant!